public/app/features/alerting/unified/RuleViewer.tsx

```diff
@@ -73,7 +73,7 @@ const RuleViewer = (): JSX.Element => {
   }
 
   // we should never get to this state
-  return <></>;
+  return <>{null}</>;
```

Comment:
This is expecting a JSX Element to be returned - similarly, I didn't want to change the component just for the sake of the eslint rule